.gitignore

```diff
@@ -1,5 +1,5 @@
 # config
```

Comment:
This could be cleaned up more, but it's not a high priority now.

Reply:
Yeah, I cleaned up a little but is basically the python default + ignoring csvs and configs.